src/analytics.ts

```diff
@@ -209,5 +212,6 @@ export const GTagEvent = (gtag: GTag | undefined, eventType: EventType): void =>
     default:
       return;
   }
-  gtag.event(eventType, eventPayload);
+  if (analytics != null)
+    firebaseLogEvent(analytics, eventType, eventPayload), console.log('logged');
```

Comment:
Check me on this, but I think you just need to separate the console log and the firebase event with a semicolon. Otherwise, great job -- didn't even know there was so much analytics that went into CoursePlan!

Reply:
Also, I noticed some of the ansci minor files were added as part of this PR. Maybe move the changes just to be on the ansci branch?